src/StringCompression.cpp

```diff
@@ -25,22 +29,15 @@ std::string_view Compression::Result::GetStringView() const
 
 ContiguousByteBufferView Compression::Result::GetView() const
 {
-	// Get a generic std::byte * to what the variant is currently storing:
+	/* Compression::Result::Storage is a smart pointer to a std::vector containing std::bytes.
+	Assuming that the current Compression::Result::Storage attribute is populated, the vector's front() method gets the first element in the vector.
+	As a pointer is needed, obtain the reference to the first element of Compression::Result::Storage to get a compatible pointer.
+	Obtain the size of the array of bytes as this is necessary for std::string_view objects. */
+
 	return
 	{
-		std::visit([](const auto & Buffer) -> const std::byte *
-		{
-			using Variant = std::decay_t<decltype(Buffer)>;
-
-			if constexpr (std::is_same_v<Variant, Compression::Result::Static>)
-			{
-				return Buffer.data();
-			}
-			else
-			{
-				return Buffer.get();
-			}
-		}, Storage), Size
+		&(this->Storage.get()->front()),
```

Comment:
Is this a safe method to use?
I suppose that if `Storage` is not empty then this would work, but if it is then would this cause issues?